src/insights_client/tests/Makefile.am

```
@@ -1,15 +1,17 @@
 
 nodist_check_SCRIPTS = test_sed.py
+check_SCRIPTS = test_metrics.py \
```

SCRIPTS are [not distributed](https://www.gnu.org/software/automake/manual/html_node/Scripts.html) by default. The existing _test_sed.py.in_ being listed in EXTRA_DIST, I assume we want to distribute them, so they can be run by _make check_.

```suggestion
dist_check_SCRIPTS = test_metrics.py \
```